src/main/java/Echoa/InvalidEventContentException.java

```diff
@@ -0,0 +1,19 @@
+package Echoa;
+
+/**
+ * InvalidDEventContentException is a class that encapsulates errors relating to event.
+ * It extends from the class InvalidTaskContentException.
+ */
+
+public class InvalidEventContentException extends InvalidTaskContentException {
+    public InvalidEventContentException() {
+        super();
+    }
+
+    public String toString() {
+        return
+        "Event has been inputted in the wrong format.\n" +
+        "Please key in the following format:\n" +
+        "event [task name] /[start day, time and/or date] /[start day, time and/or date]";
+    }
```

Comment:
I noticed most Exception uses the `getMessage` method instead. Perhaps changing this method to follow the rest might be better.